plugins/modules/gitlab_project.py

```diff
@@ -187,7 +196,6 @@
   merge_requests_enabled:
     description:
       - If merge requests can be made or not.
-      - Possible values are true and false.
```

Comment:
The line `- O(issues_access_level) and O(issues_enabled) are mutually exclusive.` is no longer here.

Reply:
I don't understand:

This part is on `merge_requests_enabled`, and I remove the boolean unneeded information, as a initiative, since we're removing it from `issues_enabled`

Reply:
Ah, sorry, I got confused.